### ISSUE TITLE
Cloud: Update the examples for available services

### ISSUE THREAD

**Issue by srsathish92** [magento/devdocs#8052]
# Topic clarification/correction request

## Topic Link

https://devdocs.magento.com/cloud/project/services.html#service-versions

## Description

Please add steps to upgrade all the services or add the warnings or notice for each services. ex: we don't have clear documentation to upgrade node service. I believe we need to create ticket to upgrade services for staging & production (pro account only).

Found: `node` service in https://devdocs.magento.com/cloud/project/services.html#service-versions & https://github.com/magento/ece-tools/blob/develop/src/Service/Validator.php#L59

## Additional information

I tried to upgrade `node` service in integration environment like given example in the devdocs & I got below error When I push to environments.

```
E: Error parsing configuration files:
      - services.node.type: "node:11" is not a valid service type.
```

**Comment by srsathish92** [comment]
@meker12 please can you look into this. Thanks

**Comment by hguthrie** [comment]
@srsathish92  At the beginning of the Configure services topic is a large note about upgrading services in Pro environments. And at the beginning of each service configuration topic, there is a similar note regarding Pro environments. There is also a note about it in the [Magento Help Center](https://support.magento.com/hc/en-us/articles/360048603692).

**Comment by srsathish92** [comment]
@hguthrie Yes, I agreed the point. As per the given link said "Customers with Pro plan, can only self-serve the services upgrades on their Integration environment" But I can't able to upgrade "node" service in integration environment. This is the my only concern.

**Comment by hguthrie** [comment]
The services are upgraded and downgraded roughly the same. The instructions are below the Service compatibility table in the [Change service version](https://devdocs.magento.com/cloud/project/services.html#change-service-version) section. Based on your error, it seems like it is a problem specific to your environment. I am not sure what you are asking docs to do. There are no special configuration instructions to present for Node beyond what is in the [Change service version](https://devdocs.magento.com/cloud/project/services.html#change-service-version) section. If you have a solution or specific instructions regarding node, feel free to propose those changes here.

**Comment by srsathish92** [comment]
@meker12 @NadiyaS Can you please give instructions to upgrade node on integration environment in Pro account.

**Comment by NadiyaS** [comment]
hi @srsathish92 ,
if you want to upgrade service on integration environment, you need to change **version** in .magento/services.yaml file https://devdocs.magento.com/cloud/project/services.html#service-values, then just commit and push version. Appropriate version number should be found by the link you already mentioned in the [description](https://devdocs.magento.com/cloud/project/services.html#service-values)

**Comment by srsathish92** [comment]
@NadiyaS As per our internal discussion in slack, "You cannot setup nodejs on your environment. You can add it to the build phase and in the build hook add necessary script which will use it" . Can you please update the same in devdocs?

@hguthrie @meker12

**Comment by dobooth** [comment]
@bdenham

**Comment by srsathish92** [comment]
@dobooth @hguthrie Any update on this?

**Comment by hguthrie** [comment]
This issue is being evaluated internally to prepare the appropriate documentation. If this is NOT a typical or generic task, then this scenario will be forwarded to the KB team for a new article. Will update the issue once that is determined. Thanks for the reminder!
